app/content/models/event.py

```diff
@@ -86,6 +88,10 @@ def get_waiting_list(self):
     def is_past_sign_off_deadline(self):
         return today() >= self.sign_off_deadline
 
+    @property
+    def is_one_hour_before_event_start(self):
```

Comment:
Ikke noe galt med metoden, men tror det kan være lurt at vi flytter alt av slike metoder som jobber med tid ut i en Mixin feks

Reply:
har du et eksempel på dette?

Reply:
Det er bare et mønster hvor man bruker arv for å separere ansvaret. Her er en link til stack overflow med et eksempler: https://stackoverflow.com/a/547714

Du trenger ikke gjøre det i denne PR'en, men tror det kunne være en god ide å flytte ut mye av det, i hvert fall alle tidssjekkene når vi lager et event.